### PR TITLE
Feat pro 1274 pinkv2 combobox

### PR DESCRIPTION
**Pull request by ernstmul**
Adds Combobox:
image

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?
✅

### CODE REVIEW

File: v2/pink-sb/src/lib/combobox/Combobox.svelte [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+                viewBox="0 0 20 20"
+                fill="none"
+                xmlns="http://www.w3.org/2000/svg"
+            >
+                <path
+                    fill-rule="evenodd"
+                    clip-rule="evenodd"
+                    d="M5.29289 7.29289C5.68342 6.90237 6.31658 6.90237 6.70711 7.29289L10 10.5858L13.2929 7.29289C13.6834 6.90237 14.3166 6.90237 14.7071 7.29289C15.0976 7.68342 15.0976 8.31658 14.7071 8.70711L10.7071 12.7071C10.3166 13.0976 9.68342 13.0976 9.29289 12.7071L5.29289 8.70711C4.90237 8.31658 4.90237 7.68342 5.29289 7.29289Z"
+                />
+            </svg>
```
Comment (ArmanNik) [inline v2/pink-sb/src/lib/combobox/Combobox.svelte:97]: Can we use the `Icon` component here?

File: v2/pink-sb/src/lib/combobox/Combobox.svelte [hunk 2]
```diff
@@ -0,0 +1,128 @@
+<script lang="ts">
+    import type { ComboboxOption, ComboboxProps } from '$lib/combobox/index.js';
+
+    export let placeholder: ComboboxProps['placeholder'] = '';
+    export let disabled: ComboboxProps['disabled'] = false;
+    export let options: ComboboxProps['options'] = [];
+    export let label: ComboboxProps['label'] = '';
+
```
Comment (ArmanNik) [inline v2/pink-sb/src/lib/combobox/Combobox.svelte:8]: We might need the `required` state like for other inputs

**Review by ArmanNik — CHANGES_REQUESTED** [review]
The list doesn't scroll when using arrows to navigate
https://github.com/user-attachments/assets/46219e60-5862-4d51-a77b-17e3e7ee2ff3


Clicking on option doesn't select it, work only by pressing enter
https://github.com/user-attachments/assets/5ee12b56-1162-4c2b-bab1-f37ff3be4a55

File: v2/pink-sb/src/lib/combobox/Combobox.svelte (hunk 1 above)
Comment (ernstmul) [reply v2/pink-sb/src/lib/combobox/Combobox.svelte:113]: Yep we can! Didn't use that one before! Good suggestion, thanks!

File: v2/pink-sb/src/lib/combobox/Combobox.svelte (hunk 2 above)
Comment (ernstmul) [reply v2/pink-sb/src/lib/combobox/Combobox.svelte:10]: Added $$restProps to the input

**Comment by ernstmul** [comment]
https://github.com/appwrite/pink/pull/225#pullrequestreview-2368597624
These are addressed

File: v2/pink-sb/src/lib/combobox/Combobox.svelte (hunk 2 above)
Comment (ArmanNik) [reply v2/pink-sb/src/lib/combobox/Combobox.svelte:10]: Not sure if the $$restProps is enough... I think we might need to show a `*` on the label too when it's required (double check with Caio)

**Review by ArmanNik — COMMENTED** [review]
The scroll behavior is still weird... when I keep the arrow pressed the selected element goes beyond the fold and then the scroll "catches up" once I stop pressing the arrow

Also, should we group this with the other inputs? We might want to use the same wrapper (`Base`) on this component too for consistency... wdyt?

**Pull request closed without merging**